task/runner.go

```diff
@@ -75,6 +81,15 @@ func NewRunner(opts RunnerOptions) Runner {
 	if opts.TaskHandlers == nil {
 		opts.TaskHandlers = defaultTasks
 	}
+	if opts.MinTaskProcessingTime == 0 {
```

Comment:
I am missing where `0` is set as default. I see that in the default we're setting `task.MinTaskProcessingTime`, which is not zero, so the only  time where this condition would be true is if we define `0` in the params?

Reply:
It can be zero if one does not configure it when creating the options object, in which case we use the default here.